### PR TITLE
Add newsweek clone 

### PR DESCRIPTION
**Pull request by jabichris**
#### What does this PR do?
This PR consists of building a clone of the Newsweek webpage.

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Really great work so far @jabichris 
Your project is coming together very nicely :100: Please consider my comments below, make appropriate changes and request another review. I wish you all the best :+1: 

#### Navigation bar
Your navigation bar is a little buggy, the positioning of elements, and padding of the navigation are in a little twist. Consider comparing what the original site looks like(including but not limited to responsiveness) to yours and make any appropriate changes. Pay attention to the mobile versions of the navigation bar :ok_hand: 
#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

#### In the magazine, Subscribe & Newsletter
Please note that these sections is not responding as they should especially on the tablet screen size, look into it thoroughly and make the appropriate changes :+1: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @jabichris, well done:clap:. Let us take care of the following:**

**On Large Screens**
1. In the header, lets have some top padding.
2. The Sign In element is not well aligned.
image

3. In the Magazine section, images are not properly scaled.

**On Medium Screens:**
4. Reduce the height of the header to look like the original. Take care of the SIgn In element.
image
5. Check the left margin of the page
6. Images are not properly scaled in the **"In the Magazine"** section.
7. Editor's Pick should be aligned 2 per row
8. Check the right margin of the **US, World, Business ...** section

**On small screens:**
9. The order of sections in the sections before **"In the magazine"** is not maintained as it is in the original page. For example, Top story should come first. You can use css order property to achieve this.

**Other issues**
- Also, check the **US, World, Business ...**, font-weight of all article titles should be uniform.
- You have so many validation errors in your markup. You should validate your markup here https://validator.w3.org/#validate_by_upload 

**Kindly address the foregoing and submit another code review request.**

**Cheers:stars:**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2562 @@
...
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col2">
+              <div
+                class="sticky2"
+                style="position: static; left: 0; top: 0; width: auto;"
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #407, all styling should be in the CSS file

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2562 @@
...
+        </div>
+      </div>
+      <div class="page-bottom">
+        <div class="block block-nw-magazine">
+          <div class="content">
+            <div class="magazine">
+              <div class="flex-sm ai-fe row">
+                <div
+                  class="flex-sm-3"
+                  style="padding-left: 15px; padding-right: 15px;"
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #1205, all styling should be in the CSS file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2562 @@
...
+                    </div>
+                    <div class="category">
+                      <a href="/2020/02/14/issue.html">February 14 ISSUE</a>
+                    </div>
+                  </div>
+                  <a href="#" class="lk gray">See All Features</a>
+                </div>
+                <div
+                  class="flex-sm-9"
+                  style="padding-left: 15px; padding-right: 15px;"
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #1221, all styling should be in the CSS file

File: index.html [hunk 4]
```diff
@@ -0,0 +1,2562 @@
...
+                    <div class="block-title">
+                      <a href="/culture">Culture &amp; Travel</a>
+                    </div>
+                    <div class="culture-items flex-sm flex-wrap">
+                      <div class="culture-item">
+                        <div class="image">
+                          <a
+                            href="/rising-teen-star-koffee-gives-reggae-jolt-mainstream-1485471"
+                          >
+                            <picture style="padding-bottom: 66.7506%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #468, all styling should be in the CSS file

File: index.html [hunk 5]
```diff
@@ -0,0 +1,2562 @@
...
+                            <a href="social-tags/music" class="active">Music</a>
+                          </div>
+                        </div>
+                      </div>
+                      <div class="culture-item">
+                        <div class="image">
+                          <a
+                            href="/2020/02/07/choose-your-own-high-stakes-adventure-1484709.html"
+                          >
+                            <picture style="padding-bottom: 66.7506%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #500, all styling should be in the CSS file

File: index.html [hunk 6]
```diff
@@ -0,0 +1,2562 @@
...
+                      <div class="feature2">
+                        <div class="block-title">
+                          MORE STORIES
+                        </div>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a
+                              href="/donald-trump-emboldened-foreign-policy-senate-acquittal-russian-ambassador-michael-mcfaul-1486030"
+                            >
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #599, all styling should be in the CSS file

File: index.html [hunk 7]
```diff
@@ -0,0 +1,2562 @@
...
+                          <div class="summary">
+                            Michael McFaul told Newsweek president will likely
+                            interpret his Senate success as proof "he knows
+                            best."
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #621, all styling should be in the CSS file

File: index.html [hunk 8]
```diff
@@ -0,0 +1,2562 @@
...
+                          </h4>
+                          <div class="summary">
+                            A string of Olympics qualifying events have already
+                            been rescheduled.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #642, all styling should be in the CSS file

File: index.html [hunk 9]
```diff
@@ -0,0 +1,2562 @@
...
+                          <div class="summary">
+                            A 7 News and Emerson College Polling survey shows
+                            Senator Bernie Sanders with a healthy lead with
+                            former Mayor Pete Buttigieg gaining ground.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #664, all styling should be in the CSS file

File: index.html [hunk 10]
```diff
@@ -0,0 +1,2562 @@
...
+                          </h4>
+                          <div class="summary">
+                            The legendary actor, best known for "Spartacus,"
+                            died on Wednesday.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #685, all styling should be in the CSS file

File: index.html [hunk 11]
```diff
@@ -0,0 +1,2562 @@
...
+                          <div class="summary">
+                            Scientists hope their research could one day help
+                            people with Alzheimer's disease and other forms of
+                            dementia.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #707, all styling should be in the CSS file

File: index.html [hunk 12]
```diff
@@ -0,0 +1,2562 @@
...
+                            The Ronald Reagan Legacy Project's goal is to get
+                            something—anything—named for Reagan in all 3,067
+                            counties in the U.S. Obama? Well, there are some
+                            schools and a shaved ice treat. Here's why.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #731, all styling should be in the CSS file

File: index.html [hunk 13]
```diff
@@ -0,0 +1,2562 @@
...
+                          <div class="summary">
+                            Health officials said four residents were "exposed
+                            to an unimmunized, international visitor who was
+                            infectious with measles."
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #753, all styling should be in the CSS file

File: index.html [hunk 14]
```diff
@@ -0,0 +1,2562 @@
...
+                          <div class="summary">
+                            "When you have a lawless president, you have to
+                            bring that to the fore and you have to spotlight
+                            that," the House Judiciary Committee chairman said.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #775, all styling should be in the CSS file

File: index.html [hunk 15]
```diff
@@ -0,0 +1,2562 @@
...
+                          <div class="summary">
+                            The current cost to the federal government for
+                            operations in Iraq is an estimated $1.922 billion,
+                            including interest and additions to the base budget.
+                          </div>
+                        </article>
+                        <article class="clearfix">
+                          <div class="image">
+                            <a href="#">
+                              <picture style="padding-bottom: 66.6667%;">
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #797, all styling should be in the CSS file

File: index.html [hunk 16]
```diff
@@ -0,0 +1,2562 @@
...
+                  </div>
+                </div>
+              </div>
+              <div class="row bottom">
+                <article class="col-sm-3">
+                  <div class="image">
+                    <a href="#">
+                      <picture class="mapping-tablet lazysize">
+                        <img
+                          clas
```
Comment (euqueme) [inline index.html]: Remove the inline styling on line #2397, all styling should be in the CSS file

Comment (euqueme) [inline index.html]: Remove the inline styling on line #2489, all styling should be in the CSS file

File: README.md [hunk 17]
```diff
@@ -1 +1,39 @@
-# Newsweek-clone
\ No newline at end of file
+# Newsweek-clone
+
+
+> This is the  project for learning pourposes
+
+
+#### Link to live view
```
Comment (euqueme) [inline README.md]: Add a Live View of your page or remove this line (#7)

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @jabichris :handshake: 

### Thank you for doing the changes requested by the previous reviewers, your page looks really good :medal_sports:  

### there is still some work to do :hammer:  :bowing_woman: 

### Bootstrap Usage

- [x] I'm sorry this wasn't pointed out to you before by the previous reviewers but this project is all about using bootstrap, so, please remove all of these selectors  from your CSS and use the equivalent bootstrap classes in your HTML file

- padding: 0; (4 ocurrences )
- padding-top: 0; (1 ocurrence )
- margin: 0; (6 ocurrences )
- margin-left: 0; (4 ocurrences )
- margin-right: 0 (3 ocurrences )
- margin-bottom: 0 (2 ocurrences )
- color: black; (6 ocurrences in yor CSS file)
- color: white; (6 ocurrences )
- color: red; (1 ocurrence)
- background-color: white; (4 ocurrences )
- display: none; (6 ocurrences )
- display: inline; (8 ocurrences)
- display: block; (11 ocurrences)
- display: flex; (11 ocurrences )
- display: block; (6 ocurrences)
- justify-content: center; (1 ocurrence )
- justify-content: space-between; (1 ocurrence )
- text-decoration: none; (1 ocurrence)
- text-align: center; (5 ocurrences )
- text-transform: none; (1 ocurrence )
- text-transform: uppercase; (10 ocurrences )
- border-top: 0; (1 ocurrence )
- flex-direction: column; (1 ocurrence)
- font-weight: bold (2 ocurrences)
- flex-wrap: wrap; (2 ocurrence )

## Small Screens

**Navbar**

- [x] The NEWSWEEK logo should be centered vertically, yours is on the top of the navbar

- Yours
image

- Original

image

**Section Positioning**

- [x] please follow the structure for the first 4 sections in this screen size as it was mention before by the previous reviewer:

1. TOP STORY
2. MORE STORIES
3. CULTURE & TRAVEL
4. FEATURED STORIES

- Yours

image

- Original

image

**MOORE STORIES**

- [x] The image of the articles should take 60% of the screen width and the headline and content only 40%

- Yours

image

- Original

image

**OPINION**

- [x] the image of the articles should take only 25%  of the screen width and the headline and content 75%

- Yours

image

- Original

image

**LATEST NEWS**

- [x] the image of the articles should take only 60%  of the screen width and the headline only 40%

- Yours
image

- Original

image

**U.S, World, Business, Tech & Science, Culture and Sports**

- [x] The image of the aside articles of these sections should take 60% of the screen width and the headline only 40%

- Yours

image

- Original

image

**SUBSCRIBE**

- [x] the overlapping magazines have to have the same size, also the content has to be centered in the page

- Yours

image

- Original

image

**NEWSLETTER**

- [x] The NEWSLETTER section starts with an image

- Yours

image

- Original

image

## Medium Screens

**NEWSLETTER**

- [x] the image takes 25% of the page width and the content 75%, the content also should be aligned at the left 

- Yours

image

- Original

image

## Big Screens

**Editor's Pick**

- [x]  The articles should be placed in 4 columns, yours are in 2

- Yours

image

- Original

image

**SUBSCRIBE ans NEWSLETTER**

- [x] should be in the same row, SUBSCRIBE taking 70% of the screen width and NEWSLETTER 30%

- Yours

image

- Original

image

## All Screen sizes

**OPINION**

- [x] the images should have half of the size they have now

- Yours

image

- Original

image

**Footer**

- [x] the twitter logo is missing

- Yours

image

- Original

image

## General

- [x] remove all inline styling I've left comments where all inline styling was found (17 lines) :arrow_down: 

- [x] fix the 95 errors detected by the [HTML validator ](https://validator.w3.org/#validate_by_upload)

image

### That's It! :raised_hands: 

### Please :pray: make the changes :memo: and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Christian!  
### Your did amazing job so far :+1: 
But still not everything looks correctly:
* [x] Tablet version looks messed up right now, everything is displayed in one column:
image
* [ ] Next section in large version still looks not like original one, `newsletter` part is smaller then `subscribe` part:
image
Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @jabichris, you have done a great job. 👍 However, we still need some fixing in the ```Tablet mode``` as pointed out by the previous reviewer. Find comment below: 😄 
- ```In the magazine``` section need padding
padd
- ```Editor Pick``` section span 2 columns but currently, we have four columns
edit
```Original version```
edit1

Kindly make these changes and submit another code review 🙂 
Happy coding 💻 
Daniel Larbi Addo

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @jabichris, great work fixing the changes. 👍 However, ```Editors pick``` on ```Smalll screen``` should take the full width. Currently, it looks this way 🙂 
- fixnow
```Original version```
now

Kindly make this change and submit code review 😄We are almost there
Happy coding 💻

**Review by addod19 — APPROVED** [review]
## Status: Approved 🎈 

**Congratulations** @jabichris, 😄  you have really done a great job and you deserve a standing ovation 🌟 . Go ahead and merge to master and submit the project completion form. 👌 

Good luck with your next project 👍 
Happy coding

**Pull request merged**